### PR TITLE
Clarify contribution workflow for new members

### LINKED ISSUE #11610: Main Page of Repo can confuse new contributors
If a new contributor is unfamiliar with github and wishes to contribute to scala, the lack of an 'issues' tab on the main repo page can be confusing and discourage involvement. While the link to 'submit issue' can be found in the readme.md, this is not a common workflow.

### PR DESCRIPTION
**Pull request by nogurenn**
Closes scala/bug#11610

Merges the Bugs/Issues section to the more explicit How To Contribute section. I didn't expect to end up merging them, but it made sense after writing the initial statements.

Adds a list of skeletal steps showing the basic contribution workflow for scala/scala.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,22 +5,26 @@ standard library, compiler, and language spec.
 
 # How to contribute
 
+Issues and bug reports for _scala/scala_ are located in [scala/bug](https://github.com/scala/bug). This tracker is also where new contributors may find good first issues to work on.
```
Comment (SethTisue) [inline README.md]: I suggest making "good first issues" a hyperlink to https://github.com/scala/bug/labels/good%20first%20issue

File: README.md [hunk 2]
```diff
@@ -5,22 +5,26 @@ standard library, compiler, and language spec.
...
 To contribute in this repo, please open a [pull request](https://help.github.com/articles/using-pull-requests/#fork--pull) from your fork of this repository.
 
+For coordinating bigger work items, we use the [scala/scala-dev tracker](https://github.com/scala/scala-dev/issues).
+
 We do have to ask you to sign the [Scala CLA](http://www.lightbend.com/contribute/cla/scala) before we can merge any of your work, to protect its open source nature.
 
-For more information on building and developing the core of Scala, make sure to read
-the rest of this README!
+The general workflow is as follows.
+1. Find/file an issue in scala/bug.
```
Comment (SethTisue) [inline README.md]: We don't have a hard requirement that a scala/bug issue must exist before opening a PR in scala/scala. I'm not sure whether that's a nuance that's worth trying to capture here?

File: README.md [hunk 3]
```diff
@@ -5,22 +5,26 @@ standard library, compiler, and language spec.
...
+2. Fork the scala/scala repo.
+3. Write your changes in your forked repo. For coding guidelines, go [here](https://github.com/scala/scala#coding-guidelines).
```
Comment (SethTisue) [inline README.md]: s/Write your changes in/Push your changes to a branch in/

**Review by SethTisue — COMMENTED** [review]
LGTM generally — I suggested a few possible tweaks

File: README.md (hunk 2 above)
Comment (nogurenn) [reply README.md]: Actually it's a reasonable sentiment. I added a new line. Hopefully it's good.

**Comment by nogurenn** [comment]
I think it's a good idea not to trigger a new build if it's just a README change. I just don't know if it's possible, and if it's a concern that was raised before.

PS: Is it good practice to "resolve conversations" on my own, or is it deferred to the reviewers as contribution etiquette? I need guidance here

**Comment by SethTisue** [comment]
> I think it's a good idea not to trigger a new build if it's just a README change. I just don't know if it's possible, and if it's a concern that was raised before.

I found an old issue on this that nobody ever got around to doing anything about: https://github.com/scala/scabot/issues/5

Since Scabot (the thing that triggers Jenkins builds and does some other routine work like assigning default milestones) is only used in this one repo, putting further effort into improving it only occasionally seems worth it these days. I do wonder from time to time if there's some off the shelf the tool that could replace it. (Scabot dates back to 2015; I can believe there was no suitable alternative at the time, but perhaps something exists today.)

> PS: Is it good practice to "resolve conversations" on my own, or is it deferred to the reviewers as contribution etiquette? I need guidance here

It's fine either way; I don't think we're consistent about that. Feel free to use your best judgment there and not worry too much about it either way.

**Review by SethTisue — APPROVED** [review]

**Comment by nogurenn** [comment]
@SethTisue GitHub Actions appears to allow ignoring of updated files/paths via the `paths-ignore` filter. It can also call on self-hosted runners on trigger of action, and is capable of CI pipelines. I just took a quick look, but it seems convenient given that it's already on github and available even with the free plan.

**Comment by dwijnand** [comment]
Thanks for the PR!

**Comment by SethTisue** [comment]
@nogurenn ticket with discussion about the possibility of using GitHub Actions is https://github.com/scala/scala-dev/issues/507

**Pull request merged**